deploy/complete/docker-compose/docker-compose.yml

```diff
@@ -188,6 +192,6 @@ services:
       - NET_BIND_SERVICE
     read_only: true
     environment:
-      - OADB_USER
+      - OADB_USER=userdb_admin
```

Comment:
no need for admin here

Reply:
The existing convention is _user. In this case that would mean users_user. that sounded confusing, which is why I chose this. Will fix.